### DISCUSSION TITLE
Marine API Error

### DISCUSSION THREAD

**Discussion by FlorianRodot** [open-meteo/open-meteo#210] [General]
Hello, first of all I would like to congrats you for the great work ! Api is really useful.
This morning I was working on an app and I use your api for dev. Are you aware that the Marine Api is currently not working ? 

cheers,

Florian

**Comment by patrick-zippenfenig** [comment]
Hi, I just deployed a faulty version a couple of minutes ago.  A new version is will be out in 10 minutes!

**Reply by FlorianRodot** [reply]
Great ! Thank you for your fast answer !!

**Reply by patrick-zippenfenig** [reply]
API is working again!
